BDT_application/step2_BDTapplication_condor/TMVAClassificationApplication.C

```diff
@@ -662,11 +709,8 @@ TH1F* Getoutput( TString myMethodList = "", std::string input_name="",float xs=1
    theTree->SetBranchAddress( "dr_j1j2", &dr_j1j2);
    theTree->SetBranchAddress( "dr_j1j3", &dr_j1j3);
    theTree->SetBranchAddress( "dr_j2j3", &dr_j2j3);
-   theTree->SetBranchAddress( "ttc_l1_pt", &ttc_l1_pt );
-   theTree->SetBranchAddress( "ttc_l2_pt", &ttc_l2_pt);
    theTree->SetBranchAddress( "ttc_l1_eta", &ttc_l1_eta );
    theTree->SetBranchAddress( "ttc_l2_eta", &ttc_l2_eta);
-   theTree->SetBranchAddress( "ttc_mll", &ttc_mll);
```

Comment:
Hi @AlbertHsuNTUphys why did you remove this and instead put the setbranch address in every variation?

Reply:
After the discussion with @AlbertHsuNTUphys its more clear now